kubejs/server_scripts/base/entity/spawned/equipment.js

```diff
@@ -0,0 +1,87 @@
+EntityEvents.spawned((event) => {
+    if (!event.entity.isLiving()) {
+        return;
+    }
+
+    let mob = event.entity.type.split(':');
```

Comment:
Could you perhaps make variables of `mob[0]` and `mob[1]`? I don't actually know what they are from reading this :P

Reply:
Sure thing.